backend/entityservice/tests/conftest.py

```diff
@@ -163,3 +172,9 @@ def groups_project(request, requests):
     prj = create_project_response(requests, size, overlap, 'groups', encoding_size)
     yield prj
     delete_project(requests, prj)
+
+
+@pytest.fixture(
+    params=itertools.product(PROJECT_RESULT_TYPES_2P, [3, 4, 5]))
```

Comment:
there are other invalid combinations:
`PROJECT_RESULT_TYPES_2P` and 1, `PROJECT_RESULT_TYPES_NP` and 1.